src/frisby/spec.js

```diff
@@ -176,11 +176,8 @@ class FrisbySpec {
   /**
    * DELETE convenience wrapper
    */
-  del(url, params) {
-    params = params || {};
-    params.method = 'DELETE';
-
-    return this.fetch(url, params);
+  del(url, params = {}) {
```

Comment:
Thank you for PR.

Please revert to original at this line.

```js
del(url, params) {
```

Reply:
fixed 🙇